pkg/agent/controller/networkpolicy/packetin.go

```diff
@@ -125,14 +126,22 @@ func (c *Controller) storeDenyConnection(pktIn *ofctrl.PacketIn) error {
 	denyConn.FlowKey = tuple
 	denyConn.DestinationServiceAddress = tuple.DestinationAddress
 	denyConn.DestinationServicePort = tuple.DestinationPort
+	denyConn.Mark = getCTMarkValue(matchers)
+	dstSvcAddress := getCTNwDstValue(matchers)
+	dstSvcPort := getCTTpDstValue(matchers)
+	if dstSvcAddress != nil {
+		denyConn.DestinationServiceAddress = *dstSvcAddress
+	}
+	if dstSvcPort != 0 {
+		denyConn.DestinationServicePort = dstSvcPort
+	}
```

Comment:
I think these represent the `Original Direction IPv4 Destination Address` and  `Original Direction Transport Layer  Source Port`. If they do not belong to a service (by checking the CTMark), we won't fill the service information.
I think we do the same strategy for the connection in the conntrack table?
https://github.com/antrea-io/antrea/blob/181671f0f8374487fd35809f3388acb657fc724c/pkg/agent/flowexporter/connections/conntrack_linux.go#L144

Reply:
Then this is perhaps a naming thing, is there any reason why we don't call them OriginalDestinationAddress and OriginalDestinationPort? They mean the ServiceAddress and ServicePort when this is a Service connection.

Reply:
I don't know the reason why we name it DestinationServiceAddress/DestinationServicePort. I agree with you that it should be filled only when it is a service connection. I think we can either rename it or fill this field only when CTMark belongs to a service. Changes for the latter one will be easier as we don't need to rename these two fields for all the packages. What do you think?

Reply:
No, I don't. I was waiting for @tnqn opinion. If we need to rename it, I think it's better to open another PR to address this issue. Otherwise, we can fill these two fields only when the CTMark belongs to a service in this PR.

Reply:
Yes, I thought you were going to update the PR to go with the second option (omit fields if we cannot identify the packet as belonging to Service traffic). That's assuming that we don't use these fields for anything else.

Reply:
I'll address this issue. 
But I think we won't see the values for these two fields in the FlowAggregator or the ClickHouse if it is a non-service connection. I found that we reset these two fields if `DestinationServicePortName` is an empty string (non-service connection or we can't find the service info). We reset them just right before we export the records. 

https://github.com/antrea-io/antrea/blob/4a52bcafe3b56b39fccda2cd9016f66db166e65c/pkg/agent/flowexporter/exporter/exporter.go#L522
https://github.com/antrea-io/antrea/blob/4a52bcafe3b56b39fccda2cd9016f66db166e65c/pkg/agent/flowexporter/exporter/exporter.go#L538

Reply:
I talked to Yun-Tang offline. We think it makes more sense to rename the fields to `OriginalDestinationAddress` and `OriginalDestinationPort`.